kubernetes/cray-istio-deploy/values.yaml

```diff
@@ -32,7 +32,7 @@ kubectl:
 
```

Comment:
You missed this one, can you also update this to `1.24.17`?

Reply:
Rectified

Reply:
Thanks!